runners/test-generation/Dockerfile

```diff
@@ -0,0 +1,87 @@
+FROM nvidia/cuda:11.8.0-cudnn8-devel-ubuntu20.04
```

Comment:
the folder of this file is called `test-generation` but I think you meant `text-generation`

Reply:
my bad. Changed it to `text-generation`